crowdkit/metrics/data/_classification.py

```diff
@@ -26,12 +26,18 @@ def _check_answers(answers: pd.DataFrame) -> None:
 
 def _label_probability(row: pd.Series, label: Any, n_labels: int) -> float:
     """Numerator in the Bayes formula"""
-    return row['skill'] if row['label'] == label else (1.0 - row['skill']) / (n_labels - 1)
+    if row['label'] == label:
+        return cast(float, row['skill'])
+    else:
+        return (1. - cast(float, row['skill'])) / (n_labels - 1)
 
 
 def _task_consistency(row: pd.Series) -> float:
     """Posterior probability for a single task"""
-    return row[row['aggregated_label']] / row['denominator'] if row['denominator'] != 0 else 0.0
+    if row['denominator'] != 0:
+        return cast(float, row[row['aggregated_label']]) / cast(float, row['denominator'])
```

Comment:
Why we use `cast`, why not explicitly cast to float?

Reply:
My first idea was ‘why not’, but I think you're more correct.